docs/UserGuide.md

```diff
@@ -475,96 +475,130 @@ Note: Calling mark or unmark would only prompt an error for already marked or un
 
 ### Tag a module
 
-> Tag a specified module from the current list of modules in Le Tracker with descriptions
+> `tag {module_code} /tags {tag_1}[, {tag_2}[, {tag_3}, ...]]]`
```

Comment:
Seems like there's an extra `]`. This issue is repeated several times in your edit.